docs/distributions/gentoo/installation.md

```diff
@@ -1,89 +1,42 @@
-    Gentoo is currently unmaintained. Due to lack of any maintainer, the Gentoo guides shall not be updated unless we get another maintainer. If you want to be a maintainer for Gentoo, contact the [Discord](https://discord.com/invite/68MRhQu).
-
-## Hardware Requirements
-
-* Gentoo Minimal Installation ISO
-    * Requirements for this ISO: USB keyboard, mouse, ethernet adapter/Wi-Fi adapter
-    * If you don't meet the requirements, then you are required to use another distro's ISO. A recommended distro to use is Ubuntu, more specifically, [T2-Ubuntu](https://wiki.t2linux.org/distributions/ubuntu/installation/#download-the-latest-safe-release).
-        * Make sure to follow [this guide](https://wiki.gentoo.org/wiki/Installation_alternatives#Installation_from_non-Gentoo_LiveCDs) along with this guide and the Gentoo Handbook starting at step 5
-* USB-C to USB-A adapter
-* USB Flash Drive
-
 ## Install Procedure
 
-1. Partition your SSD
-
-    1. Open the Bootcamp installer and follow along until it requests for you to input a Windows ISO. This should clear space for a Linux partition because of removed APFS snapshots.
-    2. Now open up Disk Utility. Make a partition with any format. The amount of space you allocate for this Linux partition will be final, so choose wisely.
-
-2. Download a Gentoo ISO and flash it to your USB Flash Drive via [Balena Etcher](https://www.balena.io/etcher/) (you can also use dd)
-3. Ensure that Secure Boot is disabled
-
-    1. Follow the instructions from [this article](https://support.apple.com/en-us/HT208198)
-    2. Once in the Startup Security Utility, set secure boot to no security and enable external boot.
-
-4. Boot into the Live USB enviornment
-
-    1. Plug in the USB Flash Drive into your computer
-    2. Startup while holding the Option key, this will put you in the macOS Startup Manager
-    3. Select the orange EFI Boot option and press enter to boot into it. (If you're using a Ubuntu Live environment, then make sure to select the orange EFI Boot option all the way to the right)
+1. Follow the [Pre-installation](https://wiki.t2linux.org/guides/preinstall) steps.
+   1. Since there is not yet a T2 Gentoo Installation ISO, you will need to use a different ISO. We recommend the [T2-Ubuntu](https://github.com/t2linux/T2-Ubuntu/releases/latest) ISO if you want a graphical environment, or the [T2-Archiso](https://github.com/t2linux/archiso-t2/releases/latest) if you don't want to have to copy wifi firmware to the livecd.
 
-5. Follow the [Gentoo Handbook](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Disks)
+2. Boot into your chosen Live USB.
 
-    1. Instead of making a new EFI partition for Gentoo, you should instead mount `/dev/nvme0n1p1` to `/mnt/gentoo/boot/efi`
-    2. Make sure that you have deleted the partition you made ahead of time, or else you'll lose important data
+3. If you chose the Ubuntu ISO, follow the [WiFi Guide](https://wiki.t2linux.org/guides/wifi-bluetooth/) once booted.
```

Comment:
I think this should be a subpoint of step 2, since not everyone is going to use the Ubuntu ISO. Also, I thought the Ubuntu ISOs came with Wi-Fi firmware files pre-packaged?

Reply:
The official Ubuntu ISO repo says to follow the WiFi guide, so I assume it doesn't.